config/layouts.php

```diff
@@ -0,0 +1,26 @@
+<?php
+/**
```

Comment:
@nathanrice This file on its own appears to have no effect for me (all layouts are available instead of only the three in this file). Should Genesis autoload that file, or do we need to filter `genesis_initial_layouts` in the Sample functions file?